packages/ui-tray/src/Tray/index.tsx

```diff
@@ -57,23 +58,9 @@ class Tray extends Component<TrayProps> {
 
   static defaultProps = {
     open: false,
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onOpen: (event) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onClose: (event) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onDismiss: (event) => {},
-    onEnter: () => {},
-    onEntering: () => {},
-    onEntered: () => {},
-    onExit: () => {},
-    onExiting: () => {},
-    onExited: () => {},
     mountNode: null,
```

Comment:
You can remove the props that are set to `null` from `defaultProps`, they are not needed anymore.